index.js

```diff
@@ -14,8 +14,14 @@ module.exports = function (options) {
       }
       var msg = message.message
       switch (message.severity) {
-        case 1: file.warn(msg, pos); break
-        case 2: file.fail(msg, pos); break
+        case 1:
+          file.message(msg, pos)
+          break
+        case 2:
+          try {
+            file.fail(msg, pos)
+          } catch (err) {}
```

Comment:
Yeah, `file.fail` throw an error (as it’s “fatal”), but we expect that to happen (thus the try/catch), especially because ESLint uses lot’s of errors, and catching allows this plugin to lint further code blocks too!